extensions/project-descriptor.md

```diff
@@ -163,11 +168,18 @@ This table MAY contain an array of buildpacks. The schema for this table is:
 id = "<buildpack ID (optional)>"
 version = "<buildpack version (optional default=latest)>"
 uri = "<url or path to the buildpack (optional default=urn:buildpack:<id>)"
+
+[io.buildpacks.build.buildpacks.script]
+api = "<buildpack api>"
+shell = "<string (optional default=/bin/sh)>"
+inline = "<script contents>"
 ```
 
 This defines the buildpacks that a platform should use on the repo.
 
-Either an `id` or a `uri` MUST be included, but MUST NOT include both. If `uri` is provided, `version` MUST NOT be allowed.
+Either an `version`, `uri`, or `script` table MUST be included, but MUST NOT include any combination of these elements.
+
+The `api` and `inline` key MUST be defined in the `script` table. The value of the `inline` key will be used as the build script for the inline buildpack this entry represents. The value of the `api` key defines its Buildpack API compatibility, and the `shell` key defines the shell used to execute the `inline` script.
```

Comment:
I think this is the first place in the spec where we use the term "inline buildpack". Should we define it somewhere more formally (e.g. a seperate subheading, a terminology section)? It would be good to have a definition to link to when folks ask about it in the future